### PR TITLE
Fix an indexing issue to make user-by-ip on the moderation alt-accounts page fast

### PR DESCRIPTION
**Pull request by jimrandomh**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208249524791513) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/lib/collections/lwevents/views.ts [hunk 1]
```diff
@@ -63,7 +63,13 @@ LWEvents.addView("gatherTownUsers", (terms: LWEventsViewTerms) => {
 })
 
 // Index used in manual user-by-IP queries, and in some moderator UI
-ensureIndex(LWEvents, {name:1, "properties.ip":1, createdAt:1, userId:1})
+ensureCustomPgIndex(`
+  CREATE INDEX CONCURRENTLY "manual_idx__LWEvents_properties_ip"
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/lwevents/views.ts]: `CREATE INDEX CONCURRENTLY IF NOT EXISTS`?

Comment (jimrandomh) [reply packages/lesswrong/lib/collections/lwevents/views.ts]: Fixed in followup commit (along with a migration, which turns out to be necessary now)

File: packages/lesswrong/lib/collections/lwevents/views.ts [hunk 2]
```diff
@@ -63,7 +63,13 @@ LWEvents.addView("gatherTownUsers", (terms: LWEventsViewTerms) => {
...
+    ON public."LWEvents" USING gin
+    ((("properties"->>'ip')::TEXT))
+    WITH (fastupdate=True)
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/lwevents/views.ts:70]: https://www.postgresql.org/docs/15/sql-createindex.html suggests `fastupdate` is set to ON by default

Comment (jimrandomh) [reply packages/lesswrong/lib/collections/lwevents/views.ts:70]: This was created using the pgAdmin UI, which put it there, and we discussed and it didn't seem like we needed to remove it.

File: packages/lesswrong/server/resolvers/moderationResolvers.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ defineQuery({
     const loginEvents = await LWEvents.find({
       name: "login",
       "properties.ip": ipAddress,
-    }, {limit: 20}).fetch();
+    }, {limit: 100}).fetch();
```
Comment (b0b3rt) [inline packages/lesswrong/server/resolvers/moderationResolvers.ts:29]: tempted to suggest that we include a projection here to only fetch the ip & userId, though it is a bit of a micro-optimization.

File: packages/lesswrong/lib/collections/lwevents/views.ts [hunk 4]
```diff
@@ -63,7 +63,13 @@ LWEvents.addView("gatherTownUsers", (terms: LWEventsViewTerms) => {
 })
 
 // Index used in manual user-by-IP queries, and in some moderator UI
-ensureIndex(LWEvents, {name:1, "properties.ip":1, createdAt:1, userId:1})
+ensureCustomPgIndex(`
```
Comment (b0b3rt) [inline packages/lesswrong/lib/collections/lwevents/views.ts:66]: needs a `void`

**Review by b0b3rt — APPROVED** [review]

File: packages/lesswrong/lib/collections/lwevents/views.ts (hunk 2 above)
Comment (b0b3rt) [reply packages/lesswrong/lib/collections/lwevents/views.ts:70]: Sure, I guess at worst it's redundant, though now I'm wondering if we have it set to OFF by default or something.

**Pull request merged**